llvm/lib/CodeGen/StackMaps.cpp

```diff
@@ -309,11 +309,11 @@ StackMaps::parseOperand(MachineInstr::const_mop_iterator MOI,
     if (MOI->isReg()) {
       if (MOI->isKill()) {
         // There's no point in tracking a killed register. Instead, try and
-        // replace it with another register with the same value (possibly due
-        // to spilling), to increase the likelyhood off us tracking that value.
-        // YKFIXME: This is probably only a temporary fix. In the future we
-        // want to allow stackmaps to track arbitrarily many locations per live
-        // variable. Currently, we can only track two.
+        // replace it with a register where the same value has been copied to,
+        // to increase the likelyhood off us tracking that value.
+	// YKFIXME: This is likely only a temporary fix. In the future we might
```

Comment:
The indentation of this looks odd to me?

Reply:
Huh, you're right. Let me check.

Reply:
For some reason there were tabs. No idea where they came from!